dashboard/config/scripts/allthethings.script

```diff
@@ -128,3 +128,6 @@ level 'Overworld Chop Tree - allthethings'
 
 stage 'Rich long assessment'
 assessment 'PS U3 Assessment1 new'
+
+stage 'Free Response'
```

Comment:
This could be good coverage to add, but I think `free_response_submittable.feature` will have pretty good coverage for `FreeResponse` levels.  Pushing this to a future PR unless you have objections.